lib/modules/platform/bitbucket/readme.md

```diff
@@ -1,5 +1,24 @@
 # Bitbucket Cloud
 
+Most of the information on this page is meant for users who want to self-host Renovate on Bitbucket Cloud.
+
+## Easiest way to run Renovate
+
+For most users, the easiest way to get started is to install [the Mend Renovate app for Bitbucket](#).
```

Comment:
Apparently we have a Mend hosted app for Bitbucket. Can you give me the link to the installation page for Bitbucket?